### PR TITLE
Implement IDENT server.

### PR DESCRIPTION
**Pull request by sayanchowdhury**

### CODE REVIEW

**Comment by slick666** [comment]
Don't we want to squash these commits?

File: ircb/identd.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+import asyncio
+import logging
+import logging.config
+import traceback
```
Comment (slick666) [inline ircb/identd.py]: This import isn't used

File: ircb/identd.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    def connection_made(self, transport):
+        print('New client connection received')
+        self.transport = transport
+
+    def data_received(self, data):
+        asyncio.Task(self.handle_data_received(data))
+
+    @asyncio.coroutine
+    def handle_data_received(self, data):
+        print(data)
```
Comment (slick666) [inline ircb/identd.py]: instead of print shouldn't we use the logging function?

File: ircb/identd.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        no_user_tmpl = "{lport}, {rport} : ERROR : NO-USER"
```
Comment (slick666) [inline ircb/identd.py]: These constants should be capitalized

**Comment by sayanchowdhury** [comment]
@slick666 This is a PR in progress, I will update the description. I will be pushing a few more updates like implementing a command to run the server and yes, I would be squashing the commits too.

File: ircb/identd.py (hunk 2 above)
Comment (sayanchowdhury) [inline ircb/identd.py]: Yes, these print statements will be removed as I will be doing some minor changes the way we log.

**Comment by slick666** [comment]
sorry @sayanchowdhury, would you please add the work-in-progress label?

File: ircb/identd.py (hunk 3 above)
Comment (sayanchowdhury) [inline ircb/identd.py]: Okay! I will capitalize them.

File: ircb/identd.py (hunk 1 above)
Comment (sayanchowdhury) [inline ircb/identd.py]: Thanks, will remove this.

**Comment by sayanchowdhury** [comment]
Added :)

**Comment by sayanchowdhury** [comment]
:+1: for me

**Pull request merged**